### PR TITLE
Unify configs

### PR DESCRIPTION
**Pull request by adydychk**

### CODE REVIEW

**Comment by adydychk** [comment]
@Intel-Media-SDK/infrastructure-write need to discuss changes around release -> Release :)

**Comment by adydychk** [comment]
Works with: https://github.com/Intel-Media-SDK/infrastructure/pull/70

File: conf_linux_public.py [hunk 1]
```diff
@@ -91,38 +91,47 @@ def _get_api_version(repo_path):
...
-                 '--no-warn-unused-cli',
-                 '-Wno-dev -G "Unix Makefiles"',
-                 '-DWARNING_FLAGS="-Wall -Werror"',
-                 '-DCMAKE_C_FLAGS_RELEASE="-O2 -D_FORTIFY_SOURCE=2 -fstack-protector"',
-                 '-DCMAKE_CXX_FLAGS_RELEASE="-O2 -D_FORTIFY_SOURCE=2 -fstack-protector"',
-                 str(MEDIA_SDK_REPO_DIR),
-]
-cmake_string = ' '.join(cmake_command)
+if args.get('api_version') == "latest": #build with api_latest
+    cmake_command = ['cmake',
```
Comment (azhogov) [inline conf_linux_public.py]: duplication

File: conf_linux_public.py [hunk 2]
```diff
@@ -91,38 +91,47 @@ def _get_api_version(repo_path):
...
-cmake_command = ['cmake',
-                 '--no-warn-unused-cli',
-                 '-Wno-dev -G "Unix Makefiles"',
-                 '-DWARNING_FLAGS="-Wall -Werror"',
-                 '-DCMAKE_C_FLAGS_RELEASE="-O2 -D_FORTIFY_SOURCE=2 -fstack-protector"',
-                 '-DCMAKE_CXX_FLAGS_RELEASE="-O2 -D_FORTIFY_SOURCE=2 -fstack-protector"',
-                 str(MEDIA_SDK_REPO_DIR),
-]
-cmake_string = ' '.join(cmake_command)
+if args.get('api_version') == "latest": #build with api_latest
```
Comment (azhogov) [inline conf_linux_public.py]: if args.get('api_latest'):

File: conf_linux_public.py [hunk 3]
```diff
@@ -95,34 +95,36 @@ def _get_api_version(repo_path):
...
                  '-DWARNING_FLAGS="-Wall -Werror"',
+                 f'-DCMAKE_BUILD_TYPE:STRING={options["BUILD_TYPE"]}',
                  '-DCMAKE_C_FLAGS_RELEASE="-O2 -D_FORTIFY_SOURCE=2 -fstack-protector"',
                  '-DCMAKE_CXX_FLAGS_RELEASE="-O2 -D_FORTIFY_SOURCE=2 -fstack-protector"',
-                 str(MEDIA_SDK_REPO_DIR),
-]
-cmake_string = ' '.join(cmake_command)
+                ]
+
+if args.get('api_latest') == "true": #build with api_latest
```
Comment (azhogov) [inline conf_linux_public.py]: it is enough:
if args.get('api_latest'):

**Pull request merged**